mlflow/langchain/__init__.py

```diff
@@ -269,6 +269,7 @@ def load_retriever(persist_directory):
         # specified by `lc_model`. Verify that the path exists and, if so, copy it to the
         # model directory along with any other specified code modules
 
+        # TODO: bbqiu can we delete this since it's handled in _validate_and_wrap_lc_model?
```

Comment:
i think this is duplicated logic, but i could def be missing something

Reply:
I think so. Lets clean it up. Agreed.

Reply:
I can clean it up in a separate PR.

Reply:
can just delete it in this PR